homeassistant/components/point/__init__.py

```diff
@@ -117,8 +117,12 @@ def token_saver(token):
             entry, data={
                 **entry.data,
             })
-    session.update_webhook(entry.data[CONF_WEBHOOK_URL],
-                           entry.data[CONF_WEBHOOK_ID], events=['*'])
+    await hass.async_add_executor_job(
+        session.update_webhook(
+            entry.data[CONF_WEBHOOK_URL],
+            entry.data[CONF_WEBHOOK_ID],
+            events=['*'],
+        ))
```

Comment:
This yields the following error:
```
Traceback (most recent call last):
  File "home-assistant/homeassistant/config_entries.py", line 259, in async_setup
    result = await component.async_setup_entry(hass, self)
  File "home-assistant/homeassistant/components/point/__init__.py", line 97, in async_setup_entry
    await async_setup_webhook(hass, entry, session)
  File "home-assistant/homeassistant/components/point/__init__.py", line 124, in async_setup_webhook
    events=['*'],
  File "python/3.7.0/Frameworks/Python.framework/Versions/3.7/lib/python3.7/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
TypeError: 'NoneType' object is not callable
```

Reply:
Pass the function as the first argument but do not call it, i.e. remove `()`.